tuxedo/pulse/14/gen3/default.nix

```diff
@@ -1,9 +1,24 @@
-{pkgs, ...}: {
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}: {
   imports = [
     ../../../../common/cpu/amd
     ../../../../common/cpu/amd/pstate.nix
     ../../../../common/cpu/amd/raphael/igpu.nix
     ../../../../common/pc/laptop
     ../../../../common/pc/laptop/ssd
   ];
+
+  # Fixing a power-issue with older kernels.
+  # When powered off, the battery does not turn off completely.
+  # Kernel 6.8.12 fixes this,
```

Comment:
@gabyx Amazing you found the exact version. I love this!

Reply:
@Mic92 WDYT?

Reply:
Looks good.